packages/gatsby-theme-api-docs/src/hooks/use-type-locations.js

```diff
@@ -23,10 +24,6 @@ const convertComponentInMdxToTypeLocations = (data) =>
     return {
       ...typeLocations,
       [`${apiKey}__${name}`]: {
-        apiKey,
-        name,
-        slug,
-        urn,
         urlAnchorTag,
```

Comment:
`urlAnchorTag` is not a good name any more.  it's a URL now and not just an anchor.